source/cloud/cloud-integrations/cloud-slash-commands.rst

```diff
@@ -4,9 +4,9 @@ Slash Commands
 
 .. note::
 
-  This is the admin documentation for slash commands. If you're a developer looking to build an integration, see `our developer documentation <https://developers.mattermost.com/integrate/getting-started/>`__.
+  This is the admin documentation for slash commands. If you're a developer looking to build an integration, see `our developer documentation <https://developers.mattermost.com/integrate>`__.
```

Comment:
This links is "broken" (the page is empty).

Reply:
Thanks, @amyblais. Fixed via https://github.com/mattermost/docs/pull/4426